### ISSUE TITLE
add-new-bcd: filters needed

### ISSUE THREAD

**Issue by Elchi3** [openwebdocs/mdn-bcd-collector#788]
I need the same filtering that `update-bcd` has in the `add-new-bcd` script:

Right now I can do:
`npm run update-bcd -- -b firefox -r 120-120`

But not:
`npm run add-new-bcd -- -b firefox -r 120-120`

**Comment by queengooborg** [comment]
The reason that `add-new-bcd` doesn't have any such filtering is because, unlike `update-bcd`, `add-new-bcd` must collect info _all_ browsers to add a brand new entry.  We probably wouldn't want to have all other browsers set to `false` if they aren't supposed to be when adding a new entry!

Additionally, I personally think that it would be better to run `add-new-bcd` regardless of what specific browser version a feature was implemented in.  My reasoning for this is that often times, a standards spec may add a feature that has already been implemented in all browsers, or a previously non-standard spec became a standard.  If we run `add-new-bcd` based upon the version a feature was added into a browser, there's a high chance we'll end up skipping a feature, and we'll have to go back and run `add-new-bcd` without filters anyways.

**Comment by Elchi3** [comment]
> The reason that `add-new-bcd` doesn't have any such filtering is because, unlike `update-bcd`, `add-new-bcd` must collect info _all_ browsers to add a brand new entry. 

Yes, I see that it must collect info on all browsers but my use case is to create a PR for a new browser release like https://github.com/mdn/browser-compat-data/pull/21083. 

> We probably wouldn't want to have all other browsers set to `false` if they aren't supposed to be when adding a new entry!

In this case all browsers would be false but Firefox would be 120. In this scenario I'm not interested in new features that are shipping in Chrome 122 and are false in all other browsers, for example. Does that make sense?

**Comment by Elchi3** [comment]
For https://github.com/mdn/browser-compat-data/pull/21148 I had to manually filter out the new stuff in Chrome 120. Would have loved if there was a filter.

**Comment by Elchi3** [comment]
I've worked through almost all of the things that add-new-bcd proposes and I opened PRs to improve the test-builders. I think if these are merged the list of things add-new-bcd proposes should be fairly small, so maybe filters are needed then after all.
